server/routes/index.js

```diff
@@ -4,4 +4,11 @@
 
 'use strict';
 
-module.exports = [].concat(require('./base'), require('./auth'), require('./search'), require('./visits'));
+// TODO just sniff out all the files under /routes?
```

Comment:
I can try taking a look. The `node-require-directory` has some pretty solid logic, although I'm having issues with getting it to include the current directory (so we may need to put the routes into a subdirectory &mdash; `server/routes/routes/base.js` instead of `server/routes/base.js`). Then we could use this one liner in `server/routes/index.js`:

```js
var requireDir = require('node-require-directory');

module.exports = requireDir('routes');
```

I tried all sorts of variants of `requireDir('.')` and `requireDir(__dirname)` but neither worked, as I'm guessing it's hitting some weird recursive thing with the index.js trying to include itself, which is why I'm thinking it needs to have all the route files in a subdirectory.